lib/node_modules/@stdlib/array/base/linspace2d/README.md

```diff
@@ -0,0 +1,134 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2025 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# linspace2d
+
+> Generate a linearly spaced two-dimensional nested numeric array.
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var linspace2d = require( '@stdlib/array/base/linspace2d' );
+```
+
+#### linspace2d( start, stop, shape, colexicographic )
+
+Generates a linearly spaced two-dimensional nested numeric array.
+
+```javascript
+var x = linspace2d( 0, 100, [ 2, 3 ], false );
+// returns [ [ 0, 20, 40 ], [ 60, 80, 100 ] ]
+
+x = linspace2d( 0, 100, [ 2, 3 ], true );
+// returns [ [ 0, 40, 80 ], [ 20, 60, 100 ] ]
+```
+
+The function accepts the following arguments:
+
+-   **start**: first array value.
+-   **stop**: last array value.
+-   **shape**: array shape.
+-   **colexicographic**: specifies whether generated array values should be stored in colexicographic order.
+
+</section>
+
+<!-- /.usage -->
+
+<section class="notes">
+
+## Notes
+
+-   The function assumes that the product of the number of rows and columns is greater than or equal to `2`.
+
```

Comment:
```suggestion
```

The function doesn't actually make any assumptions regarding the array shape, apart from being an array-like object containing two elements.